website/scss/main.scss

```diff
@@ -873,94 +872,6 @@ border and the section separator bar */
   }
 }
 
-
```

Comment:
Why did this block get removed? Was it redundant?

Reply:
Oh Saketh said he wasn't done with the PR just yet, he's gonna finish it up soon